modules/ROOT/pages/s390.adoc

```diff
@@ -0,0 +1,70 @@
+= S/390
+:description: Description about the 390 architecture
+:keywords: s390, zarch, z11, s360, s370
+:page-category: Hardware
+:source-language: c
+
+== History
+The S/390 is part of the S/3X0 family, which includes the S/360, S/370 and S/390 - all of these were released around in 1960, 1970 and 1990 respectively.
+
+== Terminology
+A quick summary for understanding some key terminology:
+
+> Hardware Floating Point, Closely related to a `float` (but not quite)
+> Dynamic Address Translation, Paging
+> Channel Subsystem I/O, DMA I/O
+> Storage, Random Acess Memory (RAM)
+> Facility, Extensions
+> Character, 8-bits
+> Halfword, 16-bits
+> Word, 32-bits
+> Double word, 64-bits
+> Define Storage (DS on HLASM), Uninitialized data
+> Define Character (DC on HLASM), Initialized data
+> Procedure, Function
+> Dataset, File
+> Processing Unit, CPU
+> Supervisor Call, SYSCALL
+> Principles Of Operation is the main manual and "architecture-reference" document, often abbreviated as POP
+> The word "Recognized" is related to an interrupt being fired up as in: "A MC interrupt is recognized when a device fails"
```

Comment:
Format this using a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/)